domain/secret/types.go

```diff
@@ -92,3 +92,11 @@ type RotationInfo struct {
 	Revision        int
 	NextTriggerTime time.Time
 }
+
+// ExpiryInfo holds information about the expiry of a secret revision.
+type ExpiryInfo struct {
+	URI             *secrets.URI
+	Revision        int
+	RevisionID      string
```

Comment:
Given the other comment, this should be RevisionUUID for consistency.

Reply:
I thought we agreed to use ID in this layer, see all the struct in this file.